Declarative Obfuscation + symbols.map + watermark

Is it possible to disable the watermark with declarative obfuscation?

When using Name protection + sequential for the mode and the MSBUILD Nuget package, no symbols.map file is generated. If using the CLI on the same project, it is.
`ConfuserEx would rename symbols to sequential string. The obfuscated name mapping would be saved to output folder in the file "symbols.map".`

Great, thanks. The CLI build works, the symbols map and PDB are both copied to output directory but I'm not sure if that already did work and the issue was just when using the nuget package. I can't get the nuget package to work - I might be doing something wrong. The package was installed via the console with
`Install-Package Confuser.MSBuild.1.5.0-alpha-gca3cdd2479.nupkg -Source C:\users\alexg\downloads\` and the solution updated but the following shows when attempting to build in release mode.

1>  [CRITICAL] Unknown error occurred.
1>C:\Users\alexg\.nuget\packages\confuser.msbuild\2.0.0-alpha-0191\build\Confuser.MSBuild.targets(61,5): error MSB4181: The "Confuser.MSBuild.Tasks.ConfuseTask" task returned false but did not log an error.

Here is the parts from the solution.
```xml
  <ItemGroup>
    <PackageReference Include="Confuser.MSBuild">
      <Version>2.0.0-alpha-0191</Version>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>
  </ItemGroup>
  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU' ">
    <Obfuscate>false</Obfuscate>
    <ConfuserReplaceOutput>true</ConfuserReplaceOutput>
    <ConfuserSymbolFileName>symbols.map</ConfuserSymbolFileName>
  </PropertyGroup>

--- a/Confuser.MSBuild.Tasks/build/Confuser.MSBuild.targets
+++ b/Confuser.MSBuild.Tasks/build/Confuser.MSBuild.targets
@@ -11,6 +11,8 @@
     <ConfuserIntermediateOutputPath Condition="'$(ConfuserIntermediateOutputPath)' == ''">$(IntermediateOutputPath)confused\</ConfuserIntermediateOutputPath>
     <ConfuserReplaceOutput Condition="'$(ConfuserReplaceOutput)' == ''">false</ConfuserReplaceOutput>
     <ConfuserOutDir Condition="'$(ConfuserOutDir)' == ''">$(OutDir)confused\</ConfuserOutDir>
+    <ConfuserSymbolFileName Condition="'$(ConfuserSymbolFileName)' == '' And '$(ConfuserReplaceOutput)' == 'true'">$(AssemblyName).symbols.map</ConfuserSymbolFileName>
+    <ConfuserSymbolFileName Condition="'$(ConfuserSymbolFileName)' == ''">symbols.map</ConfuserSymbolFileName>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(ConfuserProject)' == ''">
@@ -24,6 +26,7 @@
       _ReplaceOutputWithConfusedAssemblies;
       _ReplaceDebugOutputWithConfusedAssemblies;
       CopyConfusedFilesToOutputDirectory;      
+      _CopySymbolMapToOutputDirectory;
     </ConfuserExDependsOn>
   </PropertyGroup>
 
@@ -96,6 +99,46 @@
     <CreateItem Include="@(_ConfusedDebugSymbolsIntermediatePath)">
       <Output TaskParameter="Include" ItemName="_DebugSymbolsIntermediatePath" />
     </CreateItem>
+  </Target>
+  
+  <Target Name="_CopySymbolMapToOutputDirectory"
+          Condition="$(DesignTimeBuild) != true And $(Obfuscate) == true"
+          DependsOnTargets="ConfuseAssembly">
+    <PropertyGroup>
+      <_SymbolsMapOutputDirectory Condition="'$(ConfuserReplaceOutput)' == 'true'">$(OutDir)</_SymbolsMapOutputDirectory>
+      <_SymbolsMapOutputDirectory Condition="'$(ConfuserReplaceOutput)' != 'true'">$(ConfuserOutDir)</_SymbolsMapOutputDirectory>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <!-- By default we're not using Hard Links to copy to the output directory, and never when building in VS -->
+      <CreateHardLinksForCopyFilesToOutputDirectoryIfPossible Condition="'$(BuildingInsideVisualStudio)' == 'true' or '$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)' == ''">false</CreateHardLinksForCopyFilesToOutputDirectoryIfPossible>
+      <CreateSymbolicLinksForCopyFilesToOutputDirectoryIfPossible Condition="'$(BuildingInsideVisualStudio)' == 'true' or '$(CreateSymbolicLinksForCopyFilesToOutputDirectoryIfPossible)' == ''">false</CreateSymbolicLinksForCopyFilesToOutputDirectoryIfPossible>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <CopyConfuserSymbolsMapToOutputDirectory Condition="'$(CopyConfuserSymbolsMapToOutputDirectory)'==''">true</CopyConfuserSymbolsMapToOutputDirectory>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_SymbolsMapSourceFiles Include="$(ConfuserIntermediateOutputPath)symbols.map" />
+      <_SymbolsMapDestinationFiles Include="$(_SymbolsMapOutputDirectory)$(ConfuserSymbolFileName)" />
+    </ItemGroup>
+
+    <!-- Copy the symbols.map file in case it exists. -->
+    <Copy
+      SourceFiles="@(_SymbolsMapSourceFiles)"
+      DestinationFiles="@(_SymbolsMapDestinationFiles)"
+      SkipUnchangedFiles="$(SkipCopyUnchangedFiles)"
+      OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
+      Retries="$(CopyRetryCount)"
+      RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)"
+      UseSymboliclinksIfPossible="$(CreateSymbolicLinksForCopyFilesToOutputDirectoryIfPossible)"
+      Condition="'$(CopyConfuserSymbolsMapToOutputDirectory)' == 'true' and '$(SkipCopyBuildProduct)' != 'true'">
+      
+      <Output TaskParameter="DestinationFiles" ItemName="FileWrites"/>
+      
+    </Copy>
   </Target>
 
   <Target Name="CopyConfusedFilesToOutputDirectory"

